FP- Atom

epocopa opened the pull request.

epocopa committed:
Atom screenshots

epocopa committed:
Atom Editor- FP

Akronix commented on Fundamentos de la Programacion/fundamentos_programacion.md:
```diff
@@ -116,6 +116,45 @@ Para terminar, en la última ventana tenemos también opciones para copiar el pr
+Cualquier versión reciente de GNU/Linux, MacOs o Windows
+### 3. Guía de instalación
+#### Debian y derivadas
+Para la instalación necesitamos descargar el .deb de [aquí](https://atom.io) y posteriormente abrir una terminal teniendo acceso al usuario administrador para poder ejecutar el siguiente comando:
```
Dado que Atom no soporta la actualización automática ni a través de la aplicación, [ni a través de repositorios (oficiales)](https://github.com/atom/atom/issues/2956) para las distintas distribuciones Linux; sería interesante añadir la opciones de instalación a través de los contenedores/paquetes snap ([hay una oficial](https://insights.ubuntu.com/2017/05/11/atom-is-now-available-as-a-snap-for-ubuntu/)) y por flatpack (aunque sin versión oficial, ver: https://github.com/atom/atom/issues/11837 y https://github.com/atom/atom/pull/13293))

epocopa replied:
Hecho, aunque también hay un [PPA no oficial](https://launchpad.net/~webupd8team/+archive/ubuntu/atom)

epocopa committed:
Soporte para snap y compilación con g++

Akronix committed:
Añadido atom a FP + pequeños cambios de estilo

Movida explicación de instalar paquete snap para Atom a su propia sección.
Quitada mención de autoría de Roberto, puesto que el fichero tiene ya varios autores y está ya registrado en git.
Quitados asteriscos (*) en títulos que no sirven.
